app/Graph.cpp

```diff
@@ -0,0 +1,22 @@
+#include "Graph/Graph.hpp"
```

Comment:
For me this whole app looks like the test. Do we really need it as a separate app?

Reply:
Deleted